modules/prefix.js

```diff
@@ -148,6 +197,11 @@ var _getPrefixedValue = function (property, value) {
 // and values.
 var prefix = function (style, mode /* 'css' or 'js' */) {
   mode = mode || 'js';
+
+  if (!ExecutionEnvironment.canUseDOM) {
+    return style;
```

Comment:
Minor nitpick: The executionenvironment will never change, so why not check at module load time and make prefix the unity function if not canUseDOM?